ai-platform-samples/xgboost_multi_node/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# We modify the original Higgs data into 1GB chunks and append to different size for benchmarking. We also generate parquet files for our testing.
+
+FROM mengdong/test-rapids:latest
```

Comment:
could you replace the image to rapids ngc image?

Reply:
sure